pkg/controller/windowsmachine/windows/service.go

```diff
@@ -1,14 +1,9 @@
 package windows
 
-// service represents a Windows service
-type service interface {
-	Name() string
```

Comment:
Writing the comment here for record, @mansikulkarni96 you don't have to change the implementation now:

The downside of having just a struct for service instead of interface is you cannot have generic methods, for example, you've to expand windowsVM interface to include a method for getting status of a particular service. In a way, we're creating a dependency on the windowsVM to start and stop service. An ideal world would be something like:
```
type service interface {
Start()
Stop()
IsRunning()
Create()
Delete()
}

type windowsVM interface {
service
}
```

Now the flow would be a bit more natural in the sense, when we call vm.StartService(), we can directly call vm.Start()/ as the windows VM embeds service interface and all the methods are available in vm.StartService.